### PR TITLE
Add common HMS props to CTAS as well; Fix vectorization error message…

### PR DESCRIPTION
**Pull request by marton-bod**
… for Parquet/Avro

### CODE REVIEW

**Comment by pvary** [comment]
We might want to add a unit test for the CTAS. What do you think @marton-bod?

File: iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java [hunk 1]
```diff
@@ -380,7 +380,8 @@ public void close() throws IOException {
         case PIG:
         case HIVE:
           // TODO implement value readers for Pig and Hive
-          throw new UnsupportedOperationException("Avro support not yet supported for Pig and Hive");
+          throw new UnsupportedOperationException("Vectorized execution is not yet supported for Iceberg avro " +
```
Comment (pvary) [inline iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java:383]: We might want to separate out this to a different PR

**Comment by marton-bod** [comment]
Ah yes, testing CTAS + truncate. Good idea

File: iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java (hunk 1 above)
Comment (marton-bod) [reply iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java]: Sure!

Comment (marton-bod) [reply iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java]: https://github.com/apache/hive/pull/2965

**Review by pvary — APPROVED** [review]

**Comment by pvary** [comment]
The error message fix could be nice to have in the Iceberg code as well

**Comment by marton-bod** [comment]
Absolutely. I will create a PR for that in apache/Iceberg too.
